web/federate.go

```diff
@@ -74,6 +74,16 @@ func (h *Handler) federation(w http.ResponseWriter, req *http.Request) {
 	}
 	sort.Sort(byName(vector))
 
+	externalLabels := h.externalLabels.Clone()
+	if _, ok := externalLabels[model.InstanceLabel]; !ok {
+		externalLabels[model.InstanceLabel] = ""
+	}
+	externalLabelnames := make([]string, 0, len(externalLabels))
```

Comment:
As we usually capitalize `LabelName` and `LabelNames` (in the `model` package), I would prefer `externalLabelNames` here for consistency.